examples/learning/helloworld.html

```diff
@@ -11,9 +11,9 @@
 <canvas id="the-canvas" style="border:1px  solid black"></canvas>
 
 <!-- for legacy browsers add compatibility.js -->
-<!--<script src="../compatibility.js"></script>-->
+<!--<script src="../../node_modules/pdfjs-dist/web/compatibility.js"></script>-->
 
-<script src="../../build/generic/build/pdf.js"></script>
+<script src="../../node_modules/pdfjs-dist/pdf.js"></script>
```

Comment:
I think this should end with `/build/pdf.js` instead. Now it throws `ReferenceError: PDFJS is not defined`.